Resources/Prototypes/SimpleStation14/Entities/Objects/Fun/toys.yml

```diff
@@ -28,3 +28,29 @@
       energy: 2
     - type: RgbLightController
       layers: [0]
+
+- type: entity
+  parent: PlushieNuke
+  id: PlushieNukeBomb
+  name: nukie plushie
+  suffix: Exploding
+  description: A stuffed toy that resembles a syndicate nuclear operative. The tag claims operatives to be purely fictitious.
+  components:
+  - type: EmitSoundOnUse
+    handle: false
+  - type: OnUseTimerTrigger
+    delay: 3.5
+    examinable: false
+    beepSound:
+      path: "/Audio/Effects/beep1.ogg"
+      params:
+        volume: 2
+    initialBeepDelay: 1
+    beepInterval: 1
+  - type: Explosive
+    explosionType: Default
+    maxIntensity: 8
+    intensitySlope: 3
+    totalIntensity: 65
+    canCreateVacuum: false
+  - type: ExplodeOnTrigger
```

Comment:
This should *probably* be in a Syndie folder? Organization gets a bit shaky there, but I'd likely lean towards separating it. @DEATHB4DEFEAT

Reply:
it's a fun toy, put it with the other `Fun/toys.yml`s :)